src/arrays/referencing_array_and_its_elements/medium/q12/SumPurchase.java

```diff
@@ -0,0 +1,27 @@
+package arrays.referencing_array_and_its_elements.medium.q12;
+
+/*
+Write a Java program that asks the user to input the cost of 10 purchases and stores those values in an array of 10
```

Comment:
Change this question so it takes another input from the user after all the prices are entered which qualifies what gets printed out, something like, sum the first x purchases, or the purchases over a certain number or under a certain number or something else, since right now the user wouldn't need to use an array even if we asked them to process 1000 prices.